Assets/Scripts/Extensions/GameObjectExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+
+namespace Bug
```

Comment:
I think namespace should be Bug.Extensions

Reply:
I thought of that, but I tend to not like it, extension methods should be easy to access, and remembering to add the using Extensions namespace slows down the process in my opinion!
I like to use namespaces to separate the code, so if I don't want to separate it, I'll go less deep in the namespace!

Reply:
Sounds good to me